tests/test_gpu_stump.py

```diff
@@ -39,6 +41,59 @@ def test_gpu_stump_int_input():
         gpu_stump(np.arange(10), 5, ignore_trivial=True)
 
 
+@cuda.jit("(f8[:, :], f8[:], i8[:], i8, b1, i8[:])")
+def _gpu_searchsorted_kernel(A, V, bfs, nlevel, is_left, IDX):
+    # A wrapper kernel for calling device function _gpu_searchsorted_left/right.
+    i = cuda.grid(1)
+    if i < A.shape[0]:
+        if is_left:
+            IDX[i] = _gpu_searchsorted_left(A[i], V[i], bfs, nlevel)
+        else:
+            IDX[i] = _gpu_searchsorted_right(A[i], V[i], bfs, nlevel)
+
+
+def test_gpu_searchsorted():
+    n = 5000
```

Comment:
Let's change this to `n = 3 * config.STUMPY_THREADS_PER_BLOCK + 1`

Reply:
I will. Btw, I was getting `NumbaPerformanceWarning` regarding `GPU under usage` (or something like that). However, since data is small, I assume it is okay to ignore that warning. I will change `n`.

Reply:
You need to add `@pytest.mark.filterwarnings("ignore", category=NumbaPerformanceWarning)` before your test function

Reply:
Got it. I should have taken a look at others. Thank!